console/console.go

```diff
@@ -109,9 +101,24 @@ func InitConsole(client client.Client, odfPort int) error {
 		return err
 	}
 	// Create core ODF Plugin
-	odfConsolePlugin := GetConsolePluginCR("odf-console", "ODF Plugin", odfPort, odfService.ObjectMeta.Name, deployment.ObjectMeta)
+	odfConsolePlugin := GetConsolePluginCR("odf-console", "ODF Plugin", odfPort, odfService.Name)
 	if err := client.Create(context.TODO(), &odfConsolePlugin); err != nil && !errors.IsAlreadyExists(err) {
 		return err
 	}
 	return nil
 }
+
+func RemoveConsole(client client.Client) error {
+	consolePlugin := consolev1alpha1.ConsolePlugin{}
+	if err := client.Get(context.TODO(), types.NamespacedName{
+		Name:      "odf-console",
+		Namespace: DEPLOYMENT_NAMESPACE,
+	}, &consolePlugin); err != nil {
```

Comment:
Handle `IsNotFound` error explicitly. Since it's not technically an error in this case.